docs/source/contributing/development.md

```diff
@@ -0,0 +1,450 @@
+# Development Guide
+
+## Setup
+
+Setup should be easy, as all our packages are isolated in their own _.venv_ maintained
+by [Poetry](https://python-poetry.org/) and for managing the "meta-venv" itself, we
+have the [invocations](../../../invocations) set of helpers controlled with the
+`inv[oke]` command based on the supported [tasks](../../../tasks.py).
+
+So the first thing to run is `inv -l` to see the available list of commands and if you
+see just one, it means that you need to run `inv install-invocations` once, right
+in the root repo directory. (the _meta_ project)
+
+Explore our helper commands with `inv <cmd> --help` in order to learn more about each
+one of them.
+
+### Virtual environment(s)
+
+In case [invoke](https://www.pyinvoke.org/) isn't installed yet on your system, you
+have to do so by either your package manager or `pip install invoke` if you're fine to
+rely on system's Python interpreter. Keep in mind that we recommend developing with
+version **3.9.13** as that's what we use with most of the robots. So is better to
+develop with the same version that is expected to be at the core of our robots. (pinned
+in their _conda.yaml_)
+
+For obtaining an env with such specific version, you can either use
+[virtualenv](https://virtualenv.pypa.io/) or [pyenv](https://github.com/pyenv/pyenv),
+but that's totally up to you.
+
+With `pyenv`, I would do the following (in the repo root):
+```console
+pyenv install 3.9.13
+pyenv local 3.9.13
+```
+and `pyenv versions` would display a nice confirmation given the locally set
+interpreter:
+```console
+* 3.9.13 (set by /Users/cmin/Repos/rpaframework/.python-version)
+```
+and `pyenv which python`:
```

Comment:
Yes, but in this context is an `pyenv` action instead of a PowerShell command, so it works:
```powershell
PS Z:\Repos> pyenv which python
C:\Users\cmin\.pyenv\pyenv-win\versions\3.9.13\python.exe
```